benches/bench.rs

```diff
@@ -286,8 +364,13 @@ fn bench_decrement_no_reply(c: &mut Criterion) {
 criterion_group!(
     benches,
     bench_get,
+    bench_get_many,
```

Comment:
nice, I like that we are building out a really good benchmark suite as we add features